### PR TITLE
Add flatpages nav menu

### LINKED ISSUE #4: Add about section
Create a new menu (Events -> /) (About -> /about/)
About section should be a page with title "About" and content editable through /admin/

### PR DESCRIPTION
**Pull request by comzeradd**
- Fixes #4

### CODE REVIEW

**Comment by comzeradd** [comment]
@johngian r?

@ppapadeas you could also have a look at the UI

**Comment by ppapadeas** [comment]
r+ for UI

File: reclama/templates/flatpages/default.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+{% block title %}
+    - {{ flatpage.title }}
+{% endblock %}
+
+{% block breadcrumb %}
+  <nav class="breadcrumbs">
+    <a href="/">Bug Sprints</a>:
+    <span>{{ flatpage.title }}</span>
+
```
Comment (johngian) [inline reclama/templates/flatpages/default.html]: Please remove line.

**Comment by johngian** [comment]
LGTM r+wc

**Pull request merged**